SubcategoriesForCAP/gap/SliceCategory.gi

```diff
@@ -646,19 +648,176 @@ BindGlobal( "CAP_INTERNAL_SLICE_CATEGORY",
             local C;
             
             C := AmbientCategory( cat );
             
+            # I: i -> 1
+            # J: j -> 1
+            #
+            #       I⊗J        λ_1
+            # i⊗j -----> 1⊗1 ----> 1
+            
             return ObjectConstructor( cat,
                            PreCompose( C,
                                    TensorProductOnMorphisms( C, UnderlyingMorphism( I ), UnderlyingMorphism( J ) ),
                                    LeftUnitor( C, BaseObject( cat ) ) ) );
+        end;
+        
+        AddTensorProductOnObjects( S, TensorProductOnObjectsInSliceOverTensorUnit );
+        
+        AddTensorProductOnMorphisms( S,
+          function( cat, phi, psi )
+            local C, left_unitor, I, J, IP, JP, source, range, morphism_datum;
+            
+            C := AmbientCategory( cat );
+            
+            left_unitor := LeftUnitor( C, BaseObject( cat ) );
+            
+            #    φ           ψ
+            # i ---> i'   j ---> j'
+            #  \    /      \    /
+            # I \  / I'   J \  / J'
+            #    vv          vv
+            #    1           1
+            
+            I := Source( phi );
+            J := Source( psi );
+            
+            IP := Range( phi );
+            JP := Range( psi );
+            
+            # I⊗J: i⊗j -> 1
+            source := TensorProductOnObjectsInSliceOverTensorUnit( cat, I, J);
+            
+            # I'⊗J': i'⊗j' -> 1
+            range := TensorProductOnObjectsInSliceOverTensorUnit( cat, IP, JP );
+            
+            # φ⊗ψ: i⊗j -> i'⊗j'
+            morphism_datum := TensorProductOnMorphisms( C, UnderlyingCell( phi ), UnderlyingCell( psi ) );
+            
+            return MorphismConstructor( cat, source, morphism_datum, range );
+            
+        end );
+        
+        AddLeftUnitor( S,
+          function( cat, I )
+            local C, id_1, source, range;
+            
+            C := AmbientCategory( cat );
+            
+            #              λ_i
+            #         1⊗i ---> i
+            #           \      /
+            # id_1 ⊗ I  \    / I
+            #             v  v
+            #              1
+            
+            id_1 := ObjectConstructor( cat, IdentityMorphism( C, BaseObject( cat ) ) );
+            
+            source := TensorProductOnObjectsInSliceOverTensorUnit( cat, id_1, I );
+            
+            range := I;
+            
+            return MorphismConstructor( cat, source, LeftUnitor( C, Source( UnderlyingMorphism( I ) ) ), range );
             
         end );
         
+        AddRightUnitor( S,
+          function( cat, I )
+            local C, id_1, source, range;
+            
+            C := AmbientCategory( cat );
+            
+            #              ρ_i
+            #         i⊗1 ---> i
+            #           \      /
+            # I ⊗ id_1  \    / I
```

Comment:
Same indentation comment as above.

Reply:
I noticed that, too, but the indentation is also broken on the already existing comments:
https://github.com/homalg-project/CategoricalTowers/blob/1aea476ddfe270e44c2340a008219afd914bcb31/SubcategoriesForCAP/gap/SliceCategory.gi#L548-L558
Locally, it is not broken for me. Can you check it locally? If it is just a GitHub issue, then I would suggest leaving it that way (it would otherwise look broken locally, at least for me).

Reply:
This might not even be VSCode related...
I changed it via vim in the terminal and now it looks fine, however, in my browser and VSCode its now broken (perhaps this was what it looked like to you). :D
And it's not a tabs vs. spaces issue. 

What does it look like for you now?

Reply:
Now all looks fine 👍